docs-markdown/src/test/suite/controllers/include-controller.test.ts

```diff
@@ -50,81 +55,88 @@ suite("Include Controller", () => {
         expect(spy).to.have.been.called();
     });
     test("isMarkdownFileCheck", async () => {
-        const filePath = resolve(__dirname, testFile);
-        await loadDocumentAndGetItReady(filePath);
         const spy = chai.spy.on(common, "isMarkdownFileCheck");
-        window.showQuickPick = () => {
-            return Promise.resolve("") as Thenable<any>;
-        };
-        const stub = sinon.stub(telemetry, "sendTelemetryData");
+        const stub = sinon.stub(glob, "glob");
+        await loadDocumentAndGetItReady(testFilePath);
         await insertInclude();
-        await sleep(extendedSleepTime);
-        stub.restore();
         expect(spy).to.have.been.called();
+        stub.restore();
     });
     test("hasValidWorkSpaceRootPath", async () => {
         const spy = chai.spy.on(common, "hasValidWorkSpaceRootPath");
-        window.showQuickPick = () => {
-            return Promise.resolve("") as Thenable<any>;
-        };
-        const stub = sinon.stub(telemetry, "sendTelemetryData");
+        const stub = sinon.stub(glob, "glob");
+        await loadDocumentAndGetItReady(testFilePath);
         await insertInclude();
-        await sleep(extendedSleepTime);
-        stub.restore();
         expect(spy).to.have.been.called();
+        stub.restore();
     });
-    test("Window NT - includeOneFileEmptyLine", async () => {
-        const editor = window.activeTextEditor!;
-        moveCursor(editor, 12, 0);
-        await insertBlankLine(editor);
-        moveCursor(editor, 12, 0); // move cursor back
-        await sleep(extendedSleepTime);
-        window.showQuickPick = () => {
+    test("Window NT - insertInclude", async () => {
```

Comment:
These 3 "Windows NT" tests are failing *nix systems (tested on Ubuntu & Mac OS), i.e. 

>   1) Include Controller
>        Window NT - includeOneFileEmptyLine:
> 
>       AssertionError: expected '[!INCLUDE [1](../../repo/includes/1.md)]' to equal '[!INCLUDE [1](../includes/1.md)]'
>       + expected - actual
> 
>       -[!INCLUDE [1](../../repo/includes/1.md)]
>       +[!INCLUDE [1](../includes/1.md)]
>       

I refactored these tests and added a test specific to each arche type `Windows_NT` and `Darwin`.